widgets/common/src/deployments/DeploymentActionsMenu.tsx

```diff
@@ -1,4 +1,6 @@
+import { find } from 'lodash';
 import type { ReactNode, ComponentProps } from 'react';
```

Comment:
It seems like this import is not being used anymore (that's why jenkins pipelines are failing)
Here is a little fix for that 🎉 

```suggestion
import type { ReactNode, ComponentProps } from 'react';
```